web/src/emoji_picker.js

```diff
@@ -257,15 +257,21 @@ function filter_emojis() {
                 for (const alias of emoji_dict.aliases) {
                     const match = search_terms.every((search_term) => alias.includes(search_term));
                     if (match) {
-                        search_results.push({...emoji_dict, emoji_name: alias});
+                        const {reaction_type} = emoji.get_emoji_details_by_name(alias);
+                        search_results.push({...emoji_dict, emoji_name: alias, reaction_type});
```

Comment:
This might be a bug fix for a `high priority`  issue -- which I was unaware of [#30439](https://github.com/zulip/zulip/issues/30439). Should I post screenshot of the changes ??